api/src/main/resources/workbench.yaml

```diff
@@ -4094,9 +4095,8 @@ definitions:
         description: The name of the Jupyter VM which generated egress, e.g. all-of-us-260-m.
         type: string
 
-  RdrExportId:
-    type: object
-    properties:
-      exportId:
+  ArrayOfLong:
```

Comment:
Please add a note here that this is only necessary because of a Swagger bug and we can remove it after we update Swagger.
https://github.com/swagger-api/swagger-codegen/issues/6745